nrf_security/src/random_ext.c

```diff
@@ -7,18 +7,30 @@
 
 #include "common.h"
 #include "nrf_cc3xx_platform_ctr_drbg.h"
+#include "nrf_cc3xx_platform_hmac_drbg.h"
 #include "psa/crypto.h"
 #include "psa/crypto_platform.h"
 
  psa_status_t mbedtls_psa_external_get_random(
     mbedtls_psa_external_random_context_t *context,
     uint8_t *output, size_t output_size, size_t *output_length)
 {
-    int ret = nrf_cc3xx_platform_ctr_drbg_get(
+    int ret = PSA_ERROR_CORRUPTION_DETECTED;
+#if defined(PSA_WANT_ALG_CTR_DRBG)
+    ret = nrf_cc3xx_platform_ctr_drbg_get(
         NULL,
         output,
         output_size,
         output_length);
+#elif defined(PSA_WANT_ALG_HMAC_DRBG)
+    ret = nrf_cc3xx_platform_hmac_drbg_get(
+        NULL,
+        output,
+        output_size,
+        output_length);
+#else
+    #error random_ext: Enable CONFIG_PSA_WANT_ALG_CTR_DRBG or CONFIG_PSA_WANT_ALG_HMAC_DRBG
```

Comment:
No " ?

And please double-check that indenting preprocessor directives is OK with the coding standard.

Reply:
The indenting in nrf_security is according to mbed TLS and TF-M coding standard

Reply:
And missing quotes will be fixed

Reply:
Fixed the quotes

Reply:
> The indenting in nrf_security is according to mbed TLS and TF-M coding standard

but this file is not copied from mbedTLS but a pure NCS file, from what I can tell.
In which I believe NCS / Zephyr coding guidelines should be followed.

Reply:
We don't differentiate the coding standard inside nrf_security. All the changes and adaptions are on mbed TLS code and/or code that works together with mbed TLS